setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# 201 and 202 are spaces after ( and before ), 501 is line length. These
+# are exceptions allowed (encouraged?) by Galaxy style guidelines.
```

Comment:
"requires" feels like a strong word here - I have interpreted the language as being maybe something more like "recommends" or "suggests"? Some people don't include the whitespace - and I their their civil disobedience. 

From Wiki:

"Whitespace around operators and inside parentheses is generally included, particularly if it helps readability."